app/models/user.rb

```diff
@@ -13,7 +13,13 @@ def set_role
     self.role ||= 1
   end
 
-  def self.get_users
-    User.where(role: 1)
+  class << self
+    def get_users
+      User.where(role: 1)
+    end
+
+    def users_with_subscriptions
```

Comment:
Using a inner joins here will automatically drop any users that don't have subscriptions